projects/mci/src/main.c

```diff
@@ -42,7 +42,6 @@ int main(void) {
   soft_timer_init();
 
   prv_setup_system_can();
-  prv_mci_storage_init(&s_mci_storage);
```

Comment:
I'm assuming this is to get it to compile?

Reply:
ignore main :P